### PR TITLE
Add multirevolution lambert's example

### PR DESCRIPTION
**Pull request by jorgepiloto**
For closing #495. The *.png image was done by me also, so no problem related to licenses. Some points on this pull request:

* The plot for the case `M=1` is a little bit strange
* In the last figure, where all the orbits are painted, would you suggest a different combination of colors?

### CODE REVIEW

**Comment by jorgepiloto** [comment]
Ups! Sorry, I forgot to include this file in a `toctree`. Solving this...

File: docs/source/jupyter.rst [hunk 1]
```diff
@@ -19,3 +19,4 @@ Jupyter notebooks
    /examples/Visualizing the SpaceX Tesla Roadster trip to Mars.ipynb
    /examples/Natural and artificial perturbations.ipynb
    /examples/Porkchops with poliastro.ipynb
+   /examples/Multirevolutions solution in Lambert's problem.ipynb
```
Comment (Juanlu001) [inline docs/source/jupyter.rst]: You will have to rename the file so it doesn't have apostrophe! It used to bring problems in the past

**Comment by jorgepiloto** [comment]
I finally get rid of that apostrophe :+1:

**Comment by Juanlu001** [comment]
Excellent! Merging :rocket:

**Pull request merged**